AppCenterAnalytics/AppCenterAnalytics/Internals/MSAnalyticsPrivate.h

```diff
@@ -19,23 +19,23 @@ NS_ASSUME_NONNULL_BEGIN
 @property(nonatomic) id<MSAnalyticsDelegate> delegate;
 
 /**
- * Tenants
+ * Transmission targets.
  */
-@property(nonatomic) NSMutableDictionary *tenants;
+@property(nonatomic) NSMutableDictionary *transmissionTargets;
 
 /**
- * Default tenant.
+ * Default transmission target.
  */
-@property(nonatomic) MSAnalyticsTenant *defaultTenant;
+@property(nonatomic) MSTransmissionTarget *defaultTransmissionTarget;
```

Comment:
I think it should stay an analytics target, probably a discussion we need with other folks.